pkg/kubectl/cmd/run_test.go

```diff
@@ -114,6 +114,23 @@ func TestGetEnv(t *testing.T) {
 	}
 }
 
+func TestGetEnvFrom(t *testing.T) {
+	test := struct {
+		input    []string
+		expected []string
+	}{
+		input:    []string{"ConfigMapRef=Foo", "SecretRef=Bar"},
+		expected: []string{"ConfigMapRef=Foo", "SecretRef=Bar"},
+	}
+	cmd := &cobra.Command{}
+	cmd.Flags().StringSlice("envFrom", test.input, "")
+
+	envFromStrings := cmdutil.GetFlagStringSlice(cmd, "envFrom")
+	if len(envFromStrings) != 2 || !reflect.DeepEqual(envFromStrings, test.expected) {
+		t.Errorf("expected: %s, saw: %s", test.expected, envFromStrings)
+	}
```

Comment:
This is just a test to get the flag from cmd, I saw there is a similar one for env, so added this one, I can remove this.